fluss-server/src/main/java/com/alibaba/fluss/server/tablet/TabletService.java

```diff
@@ -350,11 +360,17 @@ private void authorizeTable(OperationType operationType, long tableId) {
             throw new UnknownTableOrBucketException(
                     String.format("This server does not host this table ID %s.", tableId));
         }
+        authorizeTable(operationType, tablePath.getTablePath());
+    }
+
+    private void authorizeTable(OperationType operationType, TablePath tablePath) {
+        if (tablePath == null) {
+            throw new UnknownTableOrBucketException(
+                    "The table path for authorization can not be null.");
+        }
```

Comment:
The `tablePath` shouldn't be null in this stage, the nullable check has been verified in `authorizeTable(OperationType operationType, long tableId)`. Otherwise, `initWriter` will throw `UnknownTableOrBucketException` if `tablePath` is null which is unexpected.